game/src/options.rs

```diff
@@ -145,26 +145,6 @@ impl OptionsPanel {
                             ColorSchemeChoice::choices(),
                         ),
                     ]),
-                    Widget::row(vec![
```

Comment:
Removed the scale_factor options UI for now. You can still set via the `--scale_factor=3.0` flag.